src/applications/personalization/dashboard-2/components/apply-for-benefits/BenefitOfInterest.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
very simple presentation component based on the existing `ApplicationInProgress`